### PR TITLE
Extension protocol extends default driver.

### PR DESCRIPTION
**Pull request by paulirish**
This also fixes the Chrome extension since it was broken for a bit.

### CODE REVIEW

File: extension/gulpfile.babel.js [hunk 1]
```diff
@@ -115,7 +119,19 @@ gulp.task('watch', ['lint', 'babel', 'html'], () => {
     'app/_locales/**/*.json'
   ]).on('change', $.livereload.reload);
 
-  gulp.watch('app/scripts.babel/**/*.js', ['lint', 'babel']);
+  const watcher = gulp.watch([
+    '*.js',
+    'app/scripts.babel/**/*.js',
+    // These below don't appear to work just yet. Hard to debug
```
Comment (brendankenny) [inline extension/gulpfile.babel.js]: I don't know how to interpret this comment :) TODO? Or issue?

File: gatherers/manifest.js [hunk 2]
```diff
@@ -25,7 +25,7 @@ class Manifest extends Gather {
 
   static _loadFromURL(options, manifestURL) {
     if (typeof window !== 'undefined') {
-      const finalURL = (new window.URL(options.driver.url).origin) + manifestURL;
+      const finalURL = (new window.URL(options.driver.url).origin) + '/' + manifestURL;
```
Comment (brendankenny) [inline gatherers/manifest.js]: testing against `window` might get us in trouble if we aren't careful with how we stub out the environment for running devtools frontent code

**Comment by brendankenny** [comment]
need to rebase

File: extension/gulpfile.babel.js (hunk 1 above)
Comment (paulirish) [inline extension/gulpfile.babel.js]: sorted it out. the watch tasks work. :)

File: gatherers/manifest.js (hunk 2 above)
Comment (paulirish) [inline gatherers/manifest.js]: yeah this part sucks but will be fixed when the manifest fetch is resolved in #83 :)

Comment (paulirish) [inline gatherers/manifest.js]: i added a `&& 'fetch' in window` for now.

**Comment by paulirish** [comment]
rebased.

**Comment by brendankenny** [comment]
:wrench: :left_right_arrow: :car:

**Pull request merged**